packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -193,19 +596,25 @@ class _ScaffoldGeometryNotifier extends ChangeNotifier implements ValueListenabl
 
 class _ScaffoldLayout extends MultiChildLayoutDelegate {
   _ScaffoldLayout({
-    @required this.statusBarHeight,
-    @required this.bottomViewInset,
-    @required this.endPadding, // for floating action button
+    @required this.minInsets, 
     @required this.textDirection,
     @required this.geometryNotifier,
-  });
-
-  final double statusBarHeight;
-  final double bottomViewInset;
-  final double endPadding;
+    // for floating action button
+    @required this.previousFloatingActionButtonPositioner,
+    @required this.currentFloatingActionButtonPositioner,
+    @required this.floatingActionButtonMoveAnimationProgress,
+    @required this.floatingActionButtonMotionAnimator,
+  }) : assert(previousFloatingActionButtonPositioner != null), assert(currentFloatingActionButtonPositioner != null);
```

Comment:
We usually put asserts like this on separate lines

Reply:
Done.